### PR TITLE
Added config for gh-pages

### PR DESCRIPTION
**Pull request by eshantri**
### Summary of Changes

- Added the config for gh-pages to deploy the website.
- Setup GitHub actions for continuous build and deployment.
- Fixed broken links across the website.
- Setup pages for publications, people and about us.

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -45,21 +53,20 @@ module.exports = {
...
+        //   type: "docsVersionDropdown",
+        //   position: "right",
+        //   dropdownActiveClassDisabled: true,
+        //   dropdownItemsAfter: [
+        //     {
+        //       to: "/versions",
+        //       label: "All versions",
+        //     },
+        //   ],
+        // },
```
Comment (im-prakher) [inline docusaurus.config.js:66]: Remove these unnecessary comments

File: docusaurus.config.js [hunk 2]
```diff
@@ -113,22 +120,20 @@ module.exports = {
...
+            current: {
+              label: "Develop",
+              path: "docs",
+            },
           },
           onlyIncludeVersions: ["master", "current"], // ex: ["current", "1.0.0", "2.0.0"]
         },
         blog: {
           showReadingTime: true,
           // Please change this to your repo.
```
Comment (im-prakher) [inline docusaurus.config.js:134]: Same here and above

File: src/components/CardList.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+            {props.email}
+          </Link>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+const CardList = () => {
+  //   console.log(contributors);
```
Comment (im-prakher) [inline src/components/CardList.js:40]: This one too

File: src/components/CardList.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  return (
+    <div className={styles.peopleContainer}>
+      {contributors.map((person, index) => {
+        console.log(person);
+        return (
+          <PeopleCard
+            key={index}
+            {...person}
+            //   name={person.name}
```
Comment (im-prakher) [inline src/components/CardList.js:49]: Same here

File: src/components/PeopleCard.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+          {/* <small className="avatar__subtitle">{props.education}</small> */}
+          {/* <Link
+            to="#"
+            onClick={(e) => {
+              window.location = `mailto:${props.email}`;
+              e.preventDefault();
+            }}
+          >
+            {props.email}
+          </Link> */}
```
Comment (im-prakher) [inline src/components/PeopleCard.js:31]: Is this useful for future reference?

Comment (im-prakher) [inline src/components/PeopleCard.js:31]: Is this useful for future reference?

File: src/components/PeopleCard.js [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+const PeopleCard = (props) => {
+  console.log(props);
+  return (
+    <div className={clsx("card-demo", styles.card)}>
+      <div className="card">
+        <div className="card__header">
+          <div className="avatar">
+            <div className="avatar__intro">
+              <div className="avatar__name">{props.name}</div>
+              {/* <small className="avatar__subtitle">{props.role}</small> */}
```
Comment (im-prakher) [inline src/components/PeopleCard.js:14]: I think this line is redundant

**Review by im-prakher — APPROVED** [review]
Rest seems good to me

File: src/components/PeopleCard.js (hunk 5 above)
Comment (eshantri) [reply src/components/PeopleCard.js:31]: Yes, since we will display the email of the contributors.

File: src/components/PeopleCard.js (hunk 6 above)
Comment (eshantri) [reply src/components/PeopleCard.js:14]: Actually role is another property of the contributor object.

Comment (im-prakher) [reply src/components/PeopleCard.js]: Okay

**Pull request merged**